File: Atividades/ATP64/src/main/java/com/SPRING/ATP64/model/Cliente.java
package com.SPRING.ATP64.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "cliente")
public class Cliente {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Getter @Setter
    private String nome;
    @Getter @Setter
    private String sobrenome;
    @Getter @Setter
    private int id;


    
}

File: Atividades/ATP76/src/Cliente.ts
export class Cliente{
    private _nome: string;
    private _sobrenome: string;
    private _idade: string;
    private _valor: number;

    set nome(nome:string){
        this._nome;
    }
    get nome():string{
        return this._nome;
    }
    set sobrenome(nome:string){
        this._sobrenome;
    }
    get sobrenome():string{
        return this._sobrenome;
    }
    set idade(idade:string){
        this._idade;
    }
    get idade():string{
        return this._idade;
    }
    set valor(valor:number){
        this._valor;
    }
    get valor():number{
        return this._valor;
    }
}